client/my-sites/checkout/checkout-thank-you/domains/domain-mapping-thank-you.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+import { ThankYou } from 'calypso/components/thank-you';
+
+import { CALYPSO_CONTACT, SUPPORT_ROOT } from 'calypso/lib/url/support';
+import { Button } from '@automattic/components';
+import domainConnectedSuccess from 'calypso/assets/images/illustrations/domain-connected-success.svg';
```

Comment:
Two nits:
1. You don't have an `Internal dependencies` section and there are extra whitespace lines between the components
2. `@automattic/components` is technically external

Reply:
I relied too heavily on the linter... Not sure why that happened. Thanks 😄